lib/cube_browser/__init__.py

```diff
@@ -539,26 +539,33 @@ def __init__(self, plots):
 
         self._slider_by_name = {}
         self._name_by_slider_id = {}
+        if self._axis_by_name:
+            name_size = max([len(name) for name in self._axis_by_name])
+        children = []
         for axis in self._axis_by_name.values():
             if hasattr(axis, 'coord') and axis.coord.units.is_time_reference():
-                opts = [(axis.coord.units.num2date(axis.coord.points[i]), i)
-                        for i in range(axis.size)]
+                pairs = [(axis.coord.units.num2date(axis.coord.points[i]), i)
+                         for i in range(axis.size)]
             elif hasattr(axis, 'coord'):
-                opts = [(axis.coord.points[i], i) for i in range(axis.size)]
+                pairs = [(axis.coord.points[i], i) for i in range(axis.size)]
             else:
-                opts = [(i, i) for i in range(axis.size)]
-            options = OrderedDict(opts)
-            slider = ipywidgets.SelectionSlider(description=axis.name,
-                                                options=options)
+                pairs = [(i, i) for i in range(axis.size)]
+            options = OrderedDict(pairs)
+            slider = ipywidgets.SelectionSlider(options=options)
```

Comment:
Apologies for my laziness, I should have written a PR description ... tsk, tsk! :wink: